install/install.sh

```diff
@@ -261,7 +273,7 @@ check_bin_dir() {
     IFS=:
     for path in $PATH; do
       if [ "${path}" = "${bin_dir}" ]; then
-        echo 1
+        printf 1
```

Comment:
This isn’t even the same thing! (`printf 1` omits the newline afterwards)

`1` does not start with `-` and does not contain `\` so it’s safe to use with `echo`.

Reply:
The newline is not necessary.

Reply:
I don't care for the rudeness, by the way. I'm all for constructive feedback, but this isn't it. 

I made the changes due to actual failing scenarios where bash needed to be installed prior to using this install script. It was not portable. I made tests to prove it and made the script posix-compliant to resolve. It wasn't based on zealotry and it wasn't intended to be optimal.

I will abandon the PR; someone else is welcome to take a crack at it, but I don't have time to argue unnecessarily over mundane minutiae.